libs/sqlserver/langchain_sqlserver/vectorstores.py

```diff
@@ -1243,9 +1243,9 @@ def _insert_embeddings(
                     # For a query, if there is no corresponding ID,
                     # we generate a uuid and add it to the list of IDs to be returned.
                     if idx < len(ids):
-                        custom_id = ids[idx]
+                        custom_id = str(ids[idx]).upper()
                     else:
-                        ids.append(str(uuid.uuid4()))
+                        ids.append(str(uuid.uuid4()).upper())
```

Comment:
Looks like it is in python that we generate the UUID if its not provided. See here. uuid.uuid4() 
In description you mention it is the system:

`, the system generates a UUID and stores it in the DB. `

Please be specific - `the python langchain package generates a UUID and stores it in DB`